templates/CRM/Contact/Form/Task/Help/Email/id-from_email.hlp

```diff
@@ -5,6 +5,9 @@
 {crmSetting var="allow_mail_from_logged_in_contact" name="allow_mail_from_logged_in_contact"}
 {if $allow_mail_from_logged_in_contact}
   <p>{ts}By default, CiviCRM uses the primary email address of the logged in user as the FROM address when sending emails to contacts. However, users with Administer CiviCRM permission can configure one or more general email addresses that can be selected as an alternative. EXAMPLE: "Client Services" &lt;clientservices@example.org&gt;{/ts}</p>
+  {if call_user_func(array('CRM_Core_Permission','check'), 'administer CiviCRM')}
+    <p>{ts 1=$smtpUrl}Go to <a href='%1'>Settings - Outbound Mail</a> to disable the usage of the logged in contact's email address as the from email{/ts}</p>
```

Comment:
This looks fine just one translation note: I know this is copy and paste from below but to avoid future changes to the `<a>` params requiring a retranslation of the whole string the [recommended way](https://docs.civicrm.org/dev/en/latest/translation/#avoid-tags-inside-strings) is `<a %1>`, and then add the href to the ts, i.e. `{ts 1='href="$smtpUrl"'}`

I don't think it's worth fixing the ones below right now since then those strings would need retranslation.

A secondary point which we can ignore for now is the `call_user_func` really shouldn't be used in templates. It has always failed in secure mode, and in smarty 5 there's a hack for it just to get tests to pass which we should get rid of someday. We can leave it for now I'm only mentioning for future.

Reply:
Ah actually there's two problems:

First, smtpUrl is defined below this line, down on line 14. So it's blank here.

Second, the variable won't evaluate within single quotes so the above is wrong. Those same docs have a solution (in the 2nd paragraph below the linked part above), but because smtpUrl is defined below it would mean fixing that definition, and then the other links would also need fixing. So, either

1. Bite the bullet and break the translations.
2. `{capture}` a new variable and just use that here, e.g.
```
{capture assign="betterSmtpUrl"}href="{crmURL p="civicrm/admin/setting/smtp" q="reset=1"}"{/capture}
<p>{ts 1=$betterSmtpUrl}Go to <a %1>Settings - Outbound Mail</a> to disable the usage of the logged in contact's email address as the from email{/ts}</p>
```